### PR TITLE
There is still some code reorganizing to do, but it's formatted correctly now 

### PR DESCRIPTION
**Pull request by thomaswmorris**

### CODE REVIEW

**Review by AbbyGi — APPROVED** [review]
Looks good!

File: bloptools/gp/scripts/benchmark.py [hunk 1]
```diff
@@ -1,21 +1,38 @@
-
```
Comment (mrakitin) [inline bloptools/gp/scripts/benchmark.py:1]: In this module, I'd add a docstring on top of the file to explain how to run this file and what parameters need to be defined in advance (like `RE`, `db`, etc.)

File: bloptools/gp/utils.py [hunk 2]
```diff
@@ -43,104 +44,141 @@ def delay_callback(from_index, to_index):
...
+    """
     How long does it take an object to go distance $x$ with acceleration $a$ and maximum velocity $v_max$?
-    That's what this function answers. 
-    '''
-    return 2*np.sqrt(np.abs(x)/a)*(np.abs(x)<v_max**2/a)+(np.abs(x)/v_max+v_max/a)*(np.abs(x)>v_max**2/a) # it works lol
+    That's what this function answers.
+    """
+    return 2 * np.sqrt(np.abs(x) / a) * (np.abs(x) < v_max**2 / a) + (np.abs(x) / v_max + v_max / a) * (
+        np.abs(x) > v_max**2 / a
+    )  # it works lol
```
Comment (mrakitin) [inline bloptools/gp/utils.py:58]:
```suggestion
    )
```

**Review by mrakitin — COMMENTED** [review]
Looks good overall. I spotted a few minor things to address (see below).

**Review by mrakitin — APPROVED** [review]
Looks good now! We agreed the docstrings will go to a separate PR.

**Pull request merged**